packages/appeals-service-api/src/app.js

```diff
@@ -11,6 +11,11 @@ require('express-async-errors');
 require('./controllers/appeals-for-submission-to-horizon-scheduler')();
 const app = express();
 
+const https = require('https');
+const http = require('http');
+https.globalAgent = new https.Agent({ keepAlive: false });
+http.globalAgent = new http.Agent({ keepAlive: false });
```

Comment:
Might be worth a comment along with this code to explain why its there? not obvious otherwise - same with other places.